### PR TITLE
travis: elasticsearch v5 support

### PR DESCRIPTION
**Pull request by lnielsen**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -36,33 +36,50 @@ cache:
...
 
 before_install:
   - "nvm install 6; nvm use 6"
+  - "mkdir /tmp/elasticsearch"
+  - "wget -O - $ES_URL | tar xz --directory=/tmp/elasticsearch --strip-components=1"
+  - "/tmp/elasticsearch/bin/elasticsearch &"
+  - "sleep 20"
   - "travis_retry pip install --upgrade pip setuptools py"
   - "travis_retry pip install twine wheel coveralls requirements-builder"
   - "requirements-builder -e all --level=min setup.py > .travis-lowest-requirements.txt"
```
Comment (slint) [inline .travis.yml]: Shouldn't it be "requirements-builder ~~-e all~~ **-e $EXTRAS** --level=min setup.py > .travis-lowest-requirements.txt"

File: examples/data/v5/testrecords/testrecord-v1.0.0.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+  "mappings": {
+    "testrecord-v1.0.0": {
+      "properties": {
+        "control_number": {
+          "type": "string"
```
Comment (slint) [inline examples/data/v5/testrecords/testrecord-v1.0.0.json]: This is ok with ES5, but fires a warning. I'm not sure if we should explicitly change ES5 mappings to use the correct field type for `string` (`text`, unless there was a `index: not_analyzed`, in which case it should be `keyword`)

File: setup.py [hunk 3]
```diff
@@ -45,13 +45,21 @@
     'redis>=2.10.0',
 ]
 
+invenio_search_version = '1.0.0b2'
```
Comment (slint) [inline setup.py]: `1.0.0b3` when https://github.com/inveniosoftware/invenio-search/pull/123 is merged

File: examples/data/v5/testrecords/testrecord-v1.0.0.json (hunk 2 above)
Comment (nharraud) [inline examples/data/v5/testrecords/testrecord-v1.0.0.json]: should be "type": "keyword"

File: examples/data/v5/testrecords/testrecord-v1.0.0.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+        },
+        "description": {
+          "type": "string"
```
Comment (nharraud) [inline examples/data/v5/testrecords/testrecord-v1.0.0.json]: "type": "text"

File: examples/data/v5/testrecords/testrecord-v1.0.0.json [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+        },
+        "participants": {
+          "type": "long"
+        },
+        "title": {
+          "type": "string",
```
Comment (nharraud) [inline examples/data/v5/testrecords/testrecord-v1.0.0.json]: "type": "text"

File: examples/data/v5/testrecords/testrecord-v1.0.0.json [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+          "copy_to": "suggest_title"
+        },
+        "type": {
+          "type": "string"
```
Comment (nharraud) [inline examples/data/v5/testrecords/testrecord-v1.0.0.json]: "type": "text"

**Comment by slint** [comment]
@lnielsen FYI, I pushed a fix for the failing builds (the bulk index queue needed to be initialized before sending records for bulk indexing)

**Pull request merged**